### PR TITLE
Add full path with yaml filename for create-indexes

### PR DESCRIPTION
**Pull request by daspecster**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -196,7 +196,7 @@ Running System Tests
    > --key-file=$P12_CREDENTIALS_FILE
 
    # Create the indexes
-   $ gcloud preview datastore create-indexes system_tests/data/ \
+   $ gcloud preview datastore create-indexes system_tests/data/index.yaml \
```
Comment (tseaver) [inline CONTRIBUTING.rst]: @dhermes This looks correct to me, per the [docs for `create-indexes`](https://cloud.google.com/sdk/gcloud/reference/preview/datastore/create-indexes).

Comment (dhermes) [inline CONTRIBUTING.rst]: :+1:

**Comment by dhermes** [comment]
@daspecster Thanks for filing! Are you running our system tests? (I'm happy someone else is :smile: )

**Comment by daspecster** [comment]
@dhermes I did and I got them to pass to a point. I got the datastore ones to run. I haven't played with gcloud stuff before but it's pretty cool so far!

**Comment by dhermes** [comment]
Good deal!

**Pull request merged**